### DISCUSSION TITLE
Certificado de JavaScript

### DISCUSSION THREAD

**Discussion by ArtumosGOC** [codaqui/institucional#57] [Certificados]
Para o certificado de JS, pensei em um arquivo em HTML para rodar o JS,
Depois pegar o arquivo JS e dar commit no github no certificado de JS,

Vantagens:

O arquivo HTML pode ser reutilizado para outras atividades, mudando o src.

Exemplo:

<!DOCTYPE html>
< html>
< head>
< meta charset="utf-8">
<script src="Projeto1.js"></script>
< /head>
< /html>

OBS: O uso da tag < meta> na < head> seria importante para uso de caracteres especiais.

**Comment by endersonmenezes** [comment]
Achei legal @ArtumosGOC pode ser um facilitador para executar os arquivos javascript, teriamos um arquivo principal que teria vários 
```
<script src="projetos/Projeto1.js"></script>
<script src="projetos/Projeto2.js"></script>
<script src="projetos/Projeto3.js"></script>
```
